cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -667,22 +676,22 @@ func TestStaticAutoscalerRunOncePodsWithFilterOutSchedulablePodsUsesPackingFalse
 	}
 
 	// Scale up
-	readyNodeListerMock.On("List").Return([]*apiv1.Node{n1, n2, n3}, nil).Times(2) // due to initialized=false
-	allNodeListerMock.On("List").Return([]*apiv1.Node{n1, n2, n3}, nil).Once()
+	readyNodeLister.SetNodes([]*apiv1.Node{n1, n2, n3})
+	allNodeLister.SetNodes([]*apiv1.Node{n1, n2, n3})
 	scheduledPodMock.On("List").Return([]*apiv1.Pod{p1, p2, p3}, nil).Times(3) // 1 to get pods + 1 per nodegroup when building nodeInfo map
```

Comment:
Maybe migrate other listers mock to test listers too?

Reply:
It would be quite big change.